pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy

```diff
@@ -50,6 +50,23 @@ pipeline {
                 }
             }
         }
+        stage('Prepare') {
+            steps {
+                dir('pd') {
+                    container("golang") {
+                        sh label: 'pd-server', script: '[ -f bin/pd-server ] || make'
```

Comment:
Can the compilation command be modified here to not include the dashboard?

Reply:
done